Model/LdapUser.php

```diff
@@ -0,0 +1,260 @@
+<?php
+
+namespace FR3D\LdapBundle\Model;
+
+use Symfony\Component\Security\Core\User\AdvancedUserInterface;
+
+/**
+ * Simple user object.
+ */
+class LdapUser implements LdapUserInterface, AdvancedUserInterface
+{
+    use UserRoleTrait;
+
+    const ROLE_DEFAULT = 'ROLE_USER';
+
+    /**
+     * @var string
+     */
+    protected $dn;
+
+    /**
+     * @var string
+     */
+    protected $username;
+
+    /**
+     * @var string
+     */
+    protected $email;
```

Comment:
We could add the email to the LdapUserInterface but the more I think about it I agree that this should be a custom option.